neptunecontrib/versioning/__init__.py

```diff
@@ -13,3 +13,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+
+from neptunecontrib.versioning.data import *
```

Comment:
Please import only what we actually want to expose. Alternatively use `__all__`.

Reply:
Done